.github/workflows/tf-plan.yaml

```
@@ -41,15 +41,24 @@ jobs:
           owner: ${{ github.repository_owner }}
 
       - name: Create in-progress check-run
-        env:
-          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
-        run: |
-          gh api repos/${{ github.repository }}/check-runs \
-            -X POST \
-            -H "Accept: application/vnd.github+json" \
-            -f name='Terraform plan' \
-            -f head_sha="${{ inputs.commit_sha }}" \
-            -f status='in_progress'
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.generate-token.outputs.token }}
+          script: |
+            const detailsUrl = `https://github.com/${context.payload.repository.full_name}/actions/runs/${context.runId}`;
```

You can make this server-agnostic like so:
```suggestion
            const detailsUrl = `${context.serverUrl}/${context.payload.repository.full_name}/actions/runs/${context.runId}`;
```